website/blog/2025-04-08-react-native-0.79.md

```diff
@@ -0,0 +1,286 @@
+
+As part of our ongoing efforts to improve debugging, we're removing Remote JS Debugging via Chrome. This legacy debugging method was deprecated, [and moved to a runtime opt-in, in React Native 0.73](https://reactnative.dev/blog/2023/12/06/0.73-debugging-improvements-stable-symlinks#remote-javascript-debugging). Please use [React Native DevTools](https://reactnative.dev/docs/react-native-devtools) for modern and reliable debugging.
+
+This also means that React Native is no longer compatible with the [react-native-debugger](https://github.com/jhen0409/react-native-debugger) community project. For developers that want to use third party debugging extensions, such as Redux DevTools, we recommend [Expo DevTools Plugins](https://github.com/expo/dev-plugins), or integrating the standalone versions of these tools.
+
+Read more in [this dedicated post](https://github.com/react-native-community/discussions-and-proposals/discussions/872).
+
+### Internal modules updated to `export` syntax
+
+As part of modernizing our JavaScript codebase, we've updated a number of implementation modules within `react-native` to consistently use `export` syntax instead of `module.exports`.
+
+We've updated around **46 APIs** in total, which can be found in the [changelog](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0790).
+
+This change has a subtle impact on existing imports:
+
+<details>
+<summary>**Case 1: Default export**</summary>
+ 
+```diff
+  // CHANGED - require() syntax
+- const ImageBackground = require('react-native/Libraries/Image/ImageBackground');
++ const ImageBackground = require('react-native/Libraries/Image/ImageBackground').default;
+
+// Unchanged - import syntax
+import ImageBackground from 'react-native/Libraries/Image/ImageBackground';
+
+// RECOMMENDED - root import
+import {ImageBackground} from 'react-native';
+
+````
+
+</details>
+
+<details>
+
+<summary>**Case 2: Secondary exports**</summary>
+
+There are very few cases of this pattern, again unaffected when using the root `'react-native'` import.
+
+```diff
+  // Unchanged - require() syntax
+  const BlobRegistry = require('react-native/Libraries/Blob/BlobRegistry');
+
+  // Unchanged - require() syntax with destructuring
+  const {register, unregister} = require('react-native/Libraries/Blob/BlobRegistry');
+
+  // CHANGED - import syntax as single object
+- import BlobRegistry from 'react-native/Libraries/Blob/BlobRegistry';
++ import * as BlobRegistry from 'react-native/Libraries/Blob/BlobRegistry';
+
+
+  // Unchanged - import syntax with destructuring
+  import {register, unregister} from 'react-native/Libraries/Blob/BlobRegistry';
+
+  // RECOMMENDED - root import
+  import {BlobRegistry} from 'react-native';
+````
+
+</details>
+
+We expect the impact of this change to be extremely limited, particularly for projects written in TypeScript and using `import` syntax. Please check for any type errors to update your code.
+
+:::warning
```

Comment:
Weird to have a recommendation in a warning block. Warning is usually something you want to avoid...
I would probably remove the warning block completely or use the `:::tip` one that renders with a green block, as [shown in the docusaurus docs.](https://docusaurus.io/docs/markdown-features/admonitions)